tests/test_yamls/pipeline_aws.yaml

```diff
@@ -13,6 +13,7 @@ run: |
   echo SKYPILOT_TASK_ID: $SKYPILOT_TASK_ID
   echo SKYPILOT_TASK_IDS
   echo "$SKYPILOT_TASK_IDS"
+  sleep 10000
```

Comment:
unintended? Or should this also go in in this PR?

Reply:
Yes, this is required to make the `pytest tests/test_smoke.py --aws` to pass. It was a mistake from a previous PR. The `pipeline_gcp.yaml` has this `sleep 10000`, but we missed it for this file.